cluster/gce/gci/configure-helper.sh

```diff
@@ -484,7 +484,7 @@ function start-kubelet {
     flags+=" --enable-debugging-handlers=false"
     flags+=" --hairpin-mode=none"
     if [[ ! -z "${KUBELET_APISERVER:-}" && ! -z "${KUBELET_CERT:-}" && ! -z "${KUBELET_KEY:-}" ]]; then
-      flags+=" --api-servers=https://${KUBELET_APISERVER}"
+      flags+=" --api-servers=https://${MASTER_RESERVED_IP}"
```

Comment:
Switching the expected variable here will almost certainly fail on GKE smoke tests. Do we really need to change the variable name? Why not just set `KUBELET_APISERVER` to the reserved IP before passing the data into the kube-env?